skunkworks_crow/src/main/res/values/strings.xml

```diff
@@ -131,6 +131,17 @@
     <string name="odk_destination_dir_error">The destination should not be empty</string>
     <string name="location_permission_needed">Location Permission Needed</string>
     <string name="location_settings_dialog">Enable location from the settings.</string>
+    <string name="title_reset_settings">Reset Application</string>
+    <string name="reset_database"> Reset Database</string>
+    <string name="reset_pref"> Rest Preferences</string>
```

Comment:
User may not know exactly what things are stored in preferences(this is an Android concept), so it can be change to `All settings(internal settings, saved settings)` similar to that we've in collect.